File: leetcode/2457/2457.swift

class Solution
{
    func unequalTriplets(_ nums: [Int]) -> Int 
    {
        var length = nums.count
        var count = 0
        for i in 0..<length
        {
            for j in i+1..<length
            {
                for k in j+1..<length
                {
                    if nums[i] != nums[j] && nums[j] != nums[k] && nums[i] != nums[k]
                    {
                        count += 1
                    }
                }
            }
        }
        return count
    }
}

File: leetcode/2457/2457.kt
class Solution {
    fun unequalTriplets(nums: IntArray): Int 
    {
        var len = nums.size
        var count = 0
        for (i in 0..len-3)
        {
            for (j in i+1..len-2)
            {
                for (k in j+1..len-1)
                {
                    if (nums[i] != nums[j] && nums[j] != nums[k] && nums[i] != nums[k])
                    {
                        count++
                    }
                }
            }
        }
        return count
    }
}